### PR TITLE
Add CVM v6 SKUs

### PR DESCRIPTION
**Pull request by kamalca**
CVM v6 SKUs need to be added to the list of CVM supporting SKUs.

### CODE REVIEW

**Comment by kamalca** [comment]
@squirrelsc @LiliDeng I don't like this method of hard coding SKU families. I don't understand the original justification for it. Is there a better solution?

**Comment by kamalca** [comment]
I see this commit https://github.com/microsoft/lisa/commit/313cefd01a18cfa295190cd6a1ee4fa4661a3106 is where the check for CVM SKU families is added, but #2370 provides no justification for why we need to check this. We should trust the capability tagging on the SKUs and we should avoid creating more work for ourselves needing to manually update SKU family lists.

**Comment by LiliDeng** [comment]
Do you know the Capabilities names for CVM and Stateless? if yes, we can update coed to reply on them.

**Comment by LiliDeng** [comment]
> I see this commit [313cefd](https://github.com/microsoft/lisa/commit/313cefd01a18cfa295190cd6a1ee4fa4661a3106) is where the check for CVM SKU families is added, but #2370 provides no justification for why we need to check this. We should trust the capability tagging on the SKUs and we should avoid creating more work for ourselves needing to manually update SKU family lists.

Do you know the Capabilities names for CVM and Stateless? if yes, we can update coed to reply on them.

File: lisa/sut_orchestrator/azure/features.py [hunk 1]
```diff
@@ -2300,24 +2300,10 @@ def create_setting(
...
-            "standarddcasv5family",
-            "standarddcadsv5family",
-            "standardecasv5family",
-            "standardecadsv5family",
-            "standarddcev5family",
-            "standarddcedv5family",
-            "standardecev5family",
-            "standardecedv5family",
-        ]:
+        if cvm_value.casefold() == "snp":
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/features.py]: Update comments above.

Comment (kamalca) [reply lisa/sut_orchestrator/azure/features.py]: Do you just want me to remove the link to CVM overview?

Comment (squirrelsc) [reply lisa/sut_orchestrator/azure/features.py]: The original link was the source of vm size families. It looks the content was moved, so the link should be removed too.

**Comment by squirrelsc** [comment]
@LiliDeng LGTM

**Review by LiliDeng — APPROVED** [review]

**Comment by LiliDeng** [comment]
@kamalca could you please make sure you have tested this change? Thanks.

**Comment by kamalca** [comment]
> @kamalca could you please make sure you have tested this change? Thanks.

I tested SNP and TDX deployment. I am currently running all of my CVM v6 validation with this change and have seen no issues.

**Pull request merged**